driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -160,7 +167,11 @@ public MappingManager getManager() {
      * @return a query that saves {@code entity} (based on it's defined mapping).
      */
     public Statement saveQuery(T entity) {
```

Comment:
Giving the reticences expressed in that Jira ticket, I would suggest that we hold off on doing this in this PR.